patterns/2-structured/maturity-model.md

```diff
@@ -92,28 +75,26 @@ When working in host teams mistakes will automatically be widely visible. In ord
 
 ### Collaboration
 
-**Channels for Providing Feedback**
+#### Channels for Providing Feedback
 
-For silos to be reduced colleagues need to be comfortable sharing feedback openly. One easy way to support that is to use the same communication principles across hierarchies.
+For silos to be reduced colleagues need to be comfortable sharing feedback openly. One easy way to support that is to use the same communication principles across hierarchies. Ideally, you will end up with proper communication channels that are known by anyone in the organization - with channels focusing on different goals (announcements, user support, development channels, infra discussions, etc.). Some of the best practices you will establish as your InnerSource projects mature: Adoption of netiquette guidelines, and opening a proven set of standard channels (which are archived, publicly accessible, searchable) for each new InnerSource project.
```

Comment:
Why do you want to combine these two paragraphs into one? Could go either way for me.